mne/source_space.py

```diff
@@ -1348,10 +1353,10 @@ def setup_source_space(subject, fname=True, spacing='oct6', surface='white',
 
 
 @verbose
-def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
+def setup_volume_source_space(subject=None, fname=None, pos=5.0, mri=None,
```

Comment:
This function can actually be used outside of the normal FreeSurfer ecosystem. `pos=dict(...)` allows for arbitrary positions and `mri=...` *might* allow for arbitrary volumes. But you can also do `subject='sample', pos=dict(...)` and it should work. The API you propose where subject implies "use MRI" makes the discrete-source-space use case of this function harder.